A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/MFD/A32NX_NDCompass.js

```diff
@@ -156,6 +146,49 @@ class Jet_MFD_NDCompass extends Jet_NDCompass {
                         graduationGroup.appendChild(line);
                     }
                 }
+                this.courseGroup = document.createElementNS(Avionics.SVG.NS, "g");
+                this.courseGroup.setAttribute("id", "CourseInfo");
+                this.rotatingCircle.appendChild(this.courseGroup);
+                {
```

Comment:
why do we need separate block here?

Reply:
Happy to drop that, although it follows the style of all the other code around if you scroll down a bit.

Reply:
yeah, okay to keep them.